### PR TITLE
Bug fix markdown problems

### PR DESCRIPTION
**Pull request by Szymiks**
Resolves: https://github.com/Code-Poets/sheetstorm/issues/109
Now I think that markdown works better, especially text display looks much better. I've changed framework which is used and I added extensions which allow to use checkboxes, tables and few more options.

### CODE REVIEW

File: employees/models.py [hunk 1]
```diff
@@ -57,4 +57,5 @@ def work_hours_str(self):
 
     @property
     def markdown_description(self):
-        return markdown(self.description)
+        return markdown(self.description, extensions=['extra', 'sane_lists', 'wikilinks',
```
Comment (rwrzesien) [inline employees/models.py]: Reformat using black ;)

File: employees/serializers.py [hunk 2]
```diff
@@ -49,8 +49,8 @@ class Meta:
             'date',
             'project',
             'author',
-            'description',
             'work_hours',
+            'description',
```
Comment (rwrzesien) [inline employees/serializers.py]: Why this change?

File: employees/templates/employees/report_list.html [hunk 3]
```diff
@@ -23,7 +22,8 @@ <h1>{{ UI_text.PAGE_TITLE.value }}</h1>
     <form action="{% url 'custom-report-list' %}" method="POST">
         {% csrf_token %}
         {% render_form serializer template_pack='rest_framework/vertical' %}
-        <input type="submit" value="{{ UI_text.CREATE_REPORT_BUTTON.value }}">
+        <a href="https://guides.github.com/features/mastering-markdown/" target="_blank">Markdown is supported</a><br/>
+       <input type="submit" value="{{ UI_text.CREATE_REPORT_BUTTON.value }}">
```
Comment (rwrzesien) [inline employees/templates/employees/report_list.html]: Please align.

File: employees/templates/employees/report_list.html [hunk 4]
```diff
@@ -62,7 +62,7 @@ <h1>{{ UI_text.PAGE_TITLE.value }}</h1>
                     {{ report.work_hours_str }}
                 </td>
                 <td class="description-column">
-                    {{ report.markdown_description|safe|linebreaksbr }}
+                {{ report.markdown_description|safe }}
```
Comment (rwrzesien) [inline employees/templates/employees/report_list.html]: Please indent :)

File: employees/serializers.py (hunk 2 above)
Comment (Szymiks) [reply employees/serializers.py]: I changed it because  I wanted to show below description field that it is supported by Markdown. I didn't know how to resolve it better, as form is rendering by serializer so I changed it like that. Maybe there is better solution. :)

Comment (rwrzesien) [reply employees/serializers.py]: You can add `helptext` to model field, check this:
https://github.com/Code-Poets/sheetstorm/pull/87/files#diff-dd359dbd34d8fff0ddd29ecb0703e366R33

and then it gets automatically displayed when rendering whole form. But this is optional, as I guess we are going to change all our templates soon right? Just FYI that it can be achieved without having to put it at the end.

Comment (Szymiks) [reply employees/serializers.py]: Ok so I will delete this, because as you said templates will be changed so I think it is not very necessary. Thanks!

File: employees/models.py (hunk 1 above)
Comment (Szymiks) [reply employees/models.py]: Done

File: employees/templates/employees/report_list.html (hunk 3 above)
Comment (Szymiks) [reply employees/templates/employees/report_list.html]: Done

File: employees/templates/employees/report_list.html (hunk 4 above)
Comment (Szymiks) [reply employees/templates/employees/report_list.html]: Done
